Doesn't build with GHC 9.0.1

This is the build error under GHC 9.0.1 with #16 applied:

```
[20 of 26] Compiling Data.Edison.Seq.FingerSeq ( src/Data/Edison/Seq/FingerSeq.hs, dist/build/Data/Edison/Seq/FingerSeq.dyn_o )

src/Data/Edison/Seq/FingerSeq.hs:50:18: error:
    Module ‘GHC.Base’ does not export ‘unsafeCoerce#’
   |
50 | import GHC.Base (unsafeCoerce#)
   |                  ^^^^^^^^^^^^^
```

--- a/edison-core/src/Data/Edison/Seq/FingerSeq.hs
+++ b/edison-core/src/Data/Edison/Seq/FingerSeq.hs
@@ -46,7 +46,7 @@ import Data.Semigroup as SG
 import Test.QuickCheck
 
 #ifdef __GLASGOW_HASKELL__
-import GHC.Base (unsafeCoerce#)
+import GHC.Exts (unsafeCoerce#)
 #endif
 
 

